src/Microsoft.DncEng.SecretManager/Program.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Diagnostics;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Microsoft.DncEng.CommandLineLib;
+using Microsoft.DncEng.CommandLineLib.Authentication;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.DncEng.SecretManager
+{
+    public class Program : DependencyInjectedConsoleApp
+    {
+        static async Task<int> Main(string[] args)
```

Comment:
I'd personally include the access modifier here. I can never remember the rules about what things have what defaults, so it's easier to just write it down.